.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This file should be added?! If not, add `.vscode` to gitignore

Reply:
Yep, it's part of our template